src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -437,8 +437,10 @@ public partial class StreamContent : System.Net.Http.HttpContent
     public partial class StringContent : System.Net.Http.ByteArrayContent
     {
         public StringContent(string content) : base (default(byte[])) { }
+        public StringContent(string content, System.Net.Http.Headers.MediaTypeHeaderValue mediaType) : base (default(byte[])) { }
```

Comment:
I thought that for reference code that ``base()`` should never be used.

Reply:
@AraHaan I'm actually clueless on that point.  What should be used?

Reply:
Well, I see that it already uses ``base()`` so that file should be fine to do it in.

Reply:
There is a markdown doc in this repo explaining how to update the ref files automatically.

Reply:
@danmoseley Thank you for letting me know.  After I follow the instructions in updating-ref-source.md I'm getting a large number of changes.  Many of those changes I did not do.

Reply:
Are they just visual (folks didn't use the tool) or functional? Anyway, you can just revert unrelated changes.

Reply:
One is an issue I should fix.  StringContent's mediaType should be string and is currently string?.

There are also several visual differences and several differences that feel like they are functional.  For example, here are some Equals differences.   Should I attach a file of all the differences (from git diff)?

-        public bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhen(true)] System.Net.Http.HttpMethod? other) { throw null; }
-        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhen(true)] object? obj) { throw null; }
+        public bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] System.Net.Http.HttpMethod? other) { throw null; }
+        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }

Reply:
Perhaps the tool should be updated.

Reply:
The ideal, I think, is that the ref files are completely and only generated output. cc @safern for whether that's correct.

Meantime I suggest to just include relevant changes.

Reply:
I just did a commit that hopefully updated the reference assembly correctly.

Reply:
@danmoseley Was my latest commit sufficient?

Reply:
@brianmed looks right as far as I can tell, thanks. I guess @scalablecory plans to review now.

Reply:
@danmoseley Just updated.  Sorry it took a few days.